doc/OneStepSetup.md

```diff
@@ -4,63 +4,61 @@ This is a streamlined process for setting up the Pi. You'll flash a preconfigure
-    If the password does not contain a single quote character, you can enclose the entire password in single quotes, like so:
-    ```
-    export WIFIPASS='password'
-    ```
-    even if it contains other characters that might otherwise be special to bash, like \\, * and $ (but note that the \\ should still be escaped with an additional \\ in order for the password to be correctly handled)
-
-    If the password does contain a single quote, you will need to use a different syntax. E.g. if the password is `pass'word`, you would use:
-    ```
-    export WIFIPASS=$'pass\'word'
-    ```
-    and if the password contains both a single quote and a backslash, e.g. `pass'wo\rd`you'd use:
-    ```
-    export WIFIPASS=$'pass\'wo\\rd'
-    ```
-
-    Similarly if your WiFi SSID has spaces in its name, make sure they're escaped or quoted.
-
-    For example, if your SSID were
-    ```
-    Foo Bar 2.4 GHz
-    ```
-    you would use
-    ```
-    export SSID=Foo\ Bar\ 2.4\ GHz
-    ```
-    or
-    ```
-    export SSID='Foo Bar 2.4 GHz'
-    ```
-
-2. Boot it in your Pi, give it a few minutes, watching for a series of flashes (2, 3, 4, 5) and then a reboot and/or the CAM/music drives to become available on your PC/Mac. If you configured automatic music syncing, the drives won't be available on the PC/Mac until music syncing is complete. The LED flash stages during setup are:
-
-    | Stage (number of flashes)  |  Activity |
-    |---|---|
-    | 2 | Verify the requested configuration is creatable |
-    | 3 | Grab scripts to start/continue setup |
-    | 4 | Create partition and files to store camera clips/music) |
-    | 5 | Setup completed; remounting filesystems as read-only and rebooting |
-
-The Pi should be available for `ssh` at `pi@teslausb.local`, over Wifi (if automatic setup works) or USB networking (if it doesn't). It takes about 5 minutes, or more depending on network speed, etc.  The default password for user `pi@teslausb.local` is `raspberry`.
+1.  Mount the card again, and in the `boot` directory create a `teslausb_setup_variables.conf` file to export the same environment variables normally needed for manual setup (including archive info, Wifi, and push notifications (if desired).
+    A sample conf file is located in the `boot` folder on the SD card. The latest sample is also available [from GitHub](https://github.com/marcone/teslausb/blob/main-dev/pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample).
+    The sample file contains documentation and suggestions for values. >**Note** >When creating/editing the configuration file on Windows, ensure that it is saved with the correct extension. It is recommended to disable the "hide extensions for known file types" option in Windows so you can see the full file name.
```

Comment:
It did go crazy here, and it was because of the two lines with blockquote markers without spaces. Not sure why it made the downstream changes, but fixed in my next push.